docs/user/02-operation-guides/troubleshooting/03-10-503-no-access.md

```diff
@@ -10,49 +10,110 @@ This behavior might be caused by a configuration error in Istio Ingress Gateway.
+2. Check all ports used by Istio Ingress Gateway:
+
+   <!-- tabs:start -->
+
+   #### **kubectl**
+   Run the following command:
+
+   ```bash
+   kubectl exec -ti -n istio-system $(kubectl get pod -l app=istio-ingressgateway -n istio-system -o name) -c istio-proxy -- netstat -lptnu
     ```
 
-5. In the case of certificate-related issues, make sure that the `kyma-gateway-certs` and `kyma-gateway-certs-cacert` Secrets are available in the `istio-system` Namespace and that they contain proper data. Run:
+   #### **Kyma Dashboard**
+   1. Go to the `istio-system` namespace.
+   2. Navigate to the **Workloads** section on the left-hand side and select **Pods**.
+   3. Search for a Pod labeled with `app=istio-ingressgateway` and click on the Pod's name.
+   ![Search for a Pod with `app=istio-ingressgateway` label](../../../assets/search-for-istio-ingress-gateway.svg)
+   4. Scroll down to find the `Containers` section and check which ports the `istio-proxy` container uses.
+   ![Check ports used by istio-proxy](../../../assets/check-istio-proxy-ports.svg)
 
-    ```bash
+
+3. If the ports `80` and `443` are not used, check the logs of the Istio Ingress Gateway container for errors related to certificates.
+
+   <!-- tabs:start -->
+   #### **kubectl**
+   Run the following command:
+   ```bash
+   kubectl logs -n istio-system -l app=istio-ingressgateway -c istio-proxy
+   ```
+   
+   #### **Kyma Dashboard**
+   Click on the **View Logs** button.
+   ![View logs of the istio-proxy-container](../../../assets/view-istio-proxy-logs.svg)
+
+   <!-- tabs:end -->
+
+
+4. In the case of certificate-related issues, make sure that the `kyma-gateway-certs` and `kyma-gateway-certs-cacert` Secrets are available in the `istio-system` Namespace and that they contain proper data.
+
+   <!-- tabs:start -->
+   #### **kubectl**
+   Run:
+   ```bash
     kubectl get secrets -n istio-system kyma-gateway-certs -oyaml
     kubectl get secrets -n istio-system kyma-gateway-certs-cacert -oyaml
-    ```
+   ```
+
+   #### **Kyma Dashboard**
+
+   ...
+
+   <!-- tabs:end -->
+    
 
-6. To regenerate a corrupted certificate, follow the tutorial to [Set up or update a custom domain TLS certificate in Kyma](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-security/sec-01-tls-certificates-security/). If you are running Kyma provisioned through Gardener, follow the [Gardener troubleshooting guide](https://kyma-project.io/docs/kyma/latest/04-operation-guides/troubleshooting/security/sec-01-certificates-gardener/) instead.
+5. To regenerate a corrupted certificate, follow the tutorial to [Set up or update a custom domain TLS certificate in Kyma](https://kyma-project.io/docs/kyma/latest/03-tutorials/00-security/sec-01-tls-certificates-security/). If you are running Kyma provisioned through Gardener, follow the [Gardener troubleshooting guide](https://kyma-project.io/docs/kyma/latest/04-operation-guides/troubleshooting/security/sec-01-certificates-gardener/) instead.
```

Comment:
To ensure the certificate is recreated I would say, make sure that API Gateway CR have `.spec.enableKymaGateway` set to `true`, e.g.:
https://github.com/kyma-project/api-gateway/blob/876a4b108ed47ae8060f5137c7df4e25459a2d20/config/samples/operator_v1alpha1_apigateway.yaml#L6